src/helpers/channels.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  ChannelType,
+  type ChatInputCommandInteraction,
+  type DMChannel,
+  type NonThreadGuildBasedChannel,
+} from 'discord.js';
+
+import { cache, type CacheEntries } from '../core/cache';
+
+type ChannelArrayCacheKey = Pick<
+  CacheEntries,
+  'quoiFeurChannels' | 'cookieHunterChannels' | 'cookieHunterDailyLogChannels'
+>;
+
+export const addChannelInCache = async (
+  interaction: ChatInputCommandInteraction,
+  featureName: string,
```

Comment:
cool refactoring

what about having stronger typing for `featureName`, instead of a string ? an enum or even a simple union could be cool